ls1intum/Artemis
# `General`: Fix two small issues on the admin pages

### Checklist
#### General
#### Client

### Motivation and Context
During the 6.0 release candidate testing session we found two small issues on the admin pages:

1) Selecting a filter on the user management page is currently not possible and leads to an error.
2) The audit log table is not formatted correctly. See Screenshots below

Fixes #6090

### Description
1) The custom modal used for filters does not produce a result. Instead, it directly modifies attributes. Therefore, the result is undefined and it's not necessary to subscribe on it.

2) Fixed a missing `td` element in the html

### Steps for Testing
Prerequisites:
- 1 Admin

1. Navigate to the user management page
2. Play around with the filters and check that they work.
3. Verify that the issue described in #6090 is fixed (multiple pages with filters active work)
4. Navigate to the audits table and check that it looks correct (see Screenshots)

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
before:
![grafik](https://user-images.githubusercontent.com/26540346/213504955-f1dc3378-2865-49d4-ae1f-fc5999d438f7.png)

now: 
![grafik](https://user-images.githubusercontent.com/26540346/213504928-7c83f377-2086-4d60-a795-619600ea2167.png)
